src/lib/core/constants.py

```diff
@@ -23,3 +23,14 @@ class NotificationType:
 
 class CDKDeployExclusions:
     LAMBDA_ASSET_EXCLUSIONS = [".venv/", "__pycache__/"]
+
+class CDKResourceNames:
+    """ Contains 'meaningful' part of AWS resources names. 
+    Specifically, ones referred in both, tooling and monitored accounts
+    """
+    IAMROLE_EXTRACT_METRICS_LAMBDA = "extract-metrics-lambda"
+    IAMROLE_MONITORED_ACC_PUT_EVENTS = "monitored-acc-put-events"
+    IAMROLE_MONITORED_ACC_EXTRACT_METRICS = "monitored-acc-extract-metrics"
```

Comment:
Great idea about CDK resource names, it would also be useful to add Event Bus name there too, as we are working with it both in monitored and tooling stacks. We can do this later.

Reply:
Nice catch. Fully agree. Fixed.